app-backend/batch/src/main/scala/stacExport/v2/CampaignExport.scala

```diff
@@ -700,15 +709,19 @@ class CampaignStacExport(
       annotationProject: AnnotationProject,
       exportAssetTypes: Option[NonEmptyList[ExportAssetType]],
       taskStatuses: List[String]
-  ): IO[(Option[newtypes.SceneItem],
-         Map[newtypes.AnnotationProjectId, newtypes.S3URL])] = {
+  ): IO[
+    (
+        Option[newtypes.SceneItem],
+        Map[newtypes.AnnotationProjectId, newtypes.S3URL]
+    )
+  ] = {
     for {
       _ <- IO(logger.info(s"Building imagery item from tile layers"))
       maybeScene <- AnnotationProjectDao
         .getFirstScene(annotationProject.id)
         .transact(xa)
       _ <- IO {
-        logger.debug(maybeScene match {
+        logger.info(maybeScene match {
```

Comment:
@aaronxsu are you sure that it should be `info` now?

Reply:
Good catch. I will switch this back to `debug`. I changed them to `info` in testing to see more logs.